### DISCUSSION TITLE
Is there a way to download updates to works previously marked as complete?

### DISCUSSION THREAD

**Discussion by pandamega** [nianeyna/ao3downloader#170] [Q&A]
Hello again, and thank you as always for your amazing work on this program!

Occasionally an author will mark a fic complete and then end up updating it with a new chapter somewhere down the line. From what I can tell, these updates will not be downloaded by by this program, since the fic was already marked as complete. Is there a way to make the program check for and download updates for fics in this situation? I'm also wondering if this applies to works that become a series after you first download them. Will AO3 downloader know to download the other works in the series if it wasn't originally part of a series?

Thank you again!

**Comment by nianeyna** [comment]
Unfortunately, there is really no way to do either of those things cleanly. In order to know if a fic has had the total chapter count adjusted or been added to a series since you downloaded it, you have to actually go to ao3 and check on that - there's just simply no way to know based only on your local files. So, in order to know which fics in your library meet those conditions, you would have to re-scrape *every single work* in your library. That's going to be very slow... 

but it is an option! Thus, if you really want to make sure you have *all* new content:

1. use the redownload function to completely refresh your library
2. use the update functions again
3. now you're up to date... for now :wink:
